Makefile

```diff
@@ -199,6 +200,22 @@ kill_port_forwardings:
 kill_all_port_forwardings:
 	scripts/utils.sh kill_port_forwardings '$(SERVICE_NAME) $(UI_SERVICE_NAME)'
 
+deploy_service_on_ocp_cluster:
+	# service
+	DEPLOY_TARGET=ocp NAMESPACE_INDEX=$(shell bash scripts/utils.sh get_namespace_index $(NAMESPACE)) \
+		DEPLOY_TAG=$(DEPLOY_TAG) DEPLOY_MANIFEST_TAG=$(DEPLOY_MANIFEST_TAG) SERVICE_KUBECONFIG=$(SERVICE_KUBECONFIG) \
+		CLUSTER_NAME=$(CLUSTER_NAME) CLUSTER_ID=$(CLUSTER_ID) PROFILE=$(PROFILE) \
+		scripts/deploy_assisted_service.sh
+
+	# UI
+	DEPLOY_TARGET=ocp NAMESPACE_INDEX=$(shell bash scripts/utils.sh get_namespace_index $(NAMESPACE)) \
+		DEPLOY_TAG=$(DEPLOY_TAG) DEPLOY_MANIFEST_TAG=$(DEPLOY_MANIFEST_TAG) SERVICE_KUBECONFIG=$(SERVICE_KUBECONFIG) \
+		CLUSTER_NAME=$(CLUSTER_NAME) CLUSTER_ID=$(CLUSTER_ID) PROFILE=$(PROFILE) \
+		scripts/deploy_ui.sh
+
+config_etc_hosts_for_ocp_cluster:
+	discovery-infra/ocp.py --config-etc-hosts -cn $(CLUSTER_NAME) -id $(CLUSTER_ID) -ns $(NAMESPACE) --service-name $(SERVICE_NAME) --profile $(PROFILE) --deploy-target ocp $(ADDITIONAL_PARAMS)
```

Comment:
do we need all those parameters? Can we use just cluster_id?

Reply:
need those for get_assisted_service_url_by_args, as we create an assisted-service client for getting the cluster.